net/tstun/wrap.go

```diff
@@ -640,21 +635,21 @@ func (c *natFamilyConfig) String() string {
 		i++
 		return true
 	})
-	count := map[netip.Addr]int{}
-	c.dstMasqAddrs.Range(func(_ key.NodePublic, v netip.Addr) bool {
-		count[v]++
-		return true
-	})
+	// count := map[netip.Addr]int{}
```

Comment:
Let's not have commented out code here. This was to get the build passing in the patch 😅
We can track the counts when building the table and then just use the value directly here